src/v8.zig

```
@@ -2715,6 +2715,53 @@ pub const InspectorSession = struct {
         ).?;
         return RemoteObject{ .handle = remote_object };
     }
+
+    pub fn unwrapObject(self: InspectorSession, allocator: std.mem.Allocator, objectId: []const u8) !UnwrapResult {
+        const in_objectId = c.CZigString{
+            .ptr = objectId.ptr,
+            .len = objectId.len,
+        };
+        var out_error: c.CZigString = .{ .ptr = null, .len = 0 };
+        var out_value_handle: ?*c.Value = null;
+        var out_context_handle: ?*c.Context = null;
+        var out_objectGroup: c.CZigString = .{ .ptr = null, .len = 0 };
+
+        const result = c.v8_inspector__Session__unwrapObject(
+            self.handle,
+            &allocator,
+            &out_error,
+            in_objectId,
+            &out_value_handle,
+            &out_context_handle,
+            &out_objectGroup,
+        );
+        if (!result) {
+            return .{ .err = CZigStringToString(out_error) };
```

What kind of errors can this be? Like, if the only way this can fail is out of memory, is it worth capturing that as a string instead of returning an error union from the function (which would allow the caller to try/catch)

---

These are errors like `No object found for ObjectId`. I'll check if there is a clear well defined set of messages. Tho even if there is I would be hesitant to turn them into errors as v8 will be updated over time. Maybe it is fine to put `NotYetDefinedError` for any unhandled messages. I'll look into it.

---

Updated: [4477393](https://github.com/lightpanda-io/zig-v8-fork/pull/51/commits/447739329ec8a1766319ad569215f33e48e2d615)
Errors now look like:
`< {"id":18,"code":-31998,"message":"InvalidRemoteObjectId"}`